packages/css-framework/src/components/_icon.scss

```diff
@@ -1,16 +1,16 @@
 // .svg-icon {
```

Comment:
Redundant styles?

Reply:
Good spot - looks like this has been sat here for ~10 months so I'll go ahead and delete it.